Package.swift

```diff
@@ -1,26 +1,33 @@
-// swift-tools-version:5.1
+// swift-tools-version:5.5
 import PackageDescription
 
 let package = Package(
     name: "TealiumCrashModule",
     platforms: [
-        .iOS(.v9)
+        .iOS(.v9),
+        .macOS(.v10_11),
+        .tvOS(.v9),
     ],
     products: [
         .library(name: "TealiumCrashModule", targets: ["TealiumCrashModule"])
     ],
     dependencies: [
-        .package(url: "https://github.com/tealium/tealium-swift", from: "2.1.0"),
-        .package(url: "https://github.com/tealium/plcrashreporter", from: "1.5.0")
+        .package(name: "TealiumSwift", url: "https://github.com/tealium/tealium-swift", .upToNextMajor(from: "2.6.0")),
+        .package(name: "PLCrashReporter", url: "https://github.com/microsoft/plcrashreporter", .upToNextMajor(from: "1.10.0")),
```

Comment:
2.7.0 was just released, should we go ahead and just update to that?

Reply:
The version 2.6.0 is the minimum version, so 2.7.0 is still included in this release